notebooks/COS/AsnFile/AsnFile.ipynb

```diff
@@ -0,0 +1,578 @@
+    "\n",
+    "- For an in-depth manual to working with COS data and a discussion of caveats and user tips, see the [COS Data Handbook](https://hst-docs.stsci.edu/display/COSDHB/).\n",
+    "- For a detailed overview of the COS instrument, see the [COS Instrument Handbook](https://hst-docs.stsci.edu/display/COSIHB/).\n",
+    "\n",
+    "<font size=\"4 \">We'll demonstrate creating an `asn` file in two ways:\n",
+    "1. **Editing an existing `asn` file to add or remove an exposure**\n",
+    "2. **Creating an entirely new `asn` file**"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## We will import the following packages:\n",
+    "\n",
+    "- `numpy` to handle array functions\n",
+    "- `astropy.io fits` and `astropy.table Table` for accessing FITS files\n",
+    "- `glob`, `os`, and `shutil` for working with system files\n",
+    "  - `glob` helps us search for filenames\n",
+    "  - `os` and `shutil` for moving files and deleting folders, respectively\n",
+    "- `astroquery.mast Mast` and `Observations` for finding and downloading data from the [MAST](https://mast.stsci.edu/portal/Mashup/Clients/Mast/Portal.html) archive\n",
+    "- `datetime` for updating fits headers with today's date\n",
+    "\n",
+    "These python packages are installed standard with the the STScI conda distribution. For more information, see our notebook tutorial on [setting up an environment](https://github.com/spacetelescope/COS-Notebooks/blob/master/Setup/Setup.ipynb)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Import for: array manipulation\n",
+    "import numpy as np\n",
+    "\n",
+    "# Import for: reading fits files\n",
+    "from astropy.io import fits                                            \n",
+    "from astropy.table import Table\n",
+    "\n",
+    "# Import for: system files\n",
+    "import glob\n",
+    "import os\n",
+    "import shutil\n",
+    "\n",
+    "# Import for: downloading the data\n",
+    "from astroquery.mast import Observations\n",
+    "\n",
+    "# Import for: changing modification date in a fits header\n",
+    "import datetime"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## We will also define a few directories we will need:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data_dir = './data/'\n",
+    "output_dir = './output/'\n",
+    "plots_dir = output_dir + 'plots/'\n",
+    "# Make the directories in case they don't exist\n",
+    "!mkdir ./data\n",
+    "!mkdir ./output \n",
+    "!mkdir ./output/plots/"
```

Comment:
As with my comment on the previous notebook I think you want to use something more cross-platform here like `shutil` or `os.path.<various things>`.

(The `!mkdir` are particularly problematic here because those assume `mkdir` exist, which is not true for example in windows)